### DISCUSSION TITLE
Does the code take off-diagonal (coherences) Wigner distribution elements into consideration?

### DISCUSSION THREAD

**Discussion by suan12** [anharmonic/d3q#3] [General]
Dear sir,

Hello. I'm sorry to bother you. Does the code now includes both the populations’ and coherences’ contributions to thermal transport in crystals? In the previous study [Simoncelli, M., Marzari, N. & Mauri, F. Unified theory of thermal transport in crystals and glasses, Nat. Phys., 2019,15, 809–813], the authors say that the custom code developed in this work will be made available in a next release of the D3Q package. However, I did not see any description in the manual.

Looking forward to receiving your reply. Thank you very much.
Best regards,
pfliu

**Comment by paulatz** [comment]
Hello,
that code is not available yet. You may try to contact dr Simoncelli or pr Marzari directly if you need to use it. Anyway, it is a very simple term to compute once one has the phonon lifetimes. I.e. if you feel like writing some code, have a look at matlab script tools/recompute_sma.m 

regards

**Reply by suan12** [reply]
The answers are very helpful. Thanks very much!
